### ISSUE TITLE
Interactive screensharing

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/spreed#449]
Maybe you saw it, Slack just introduced more interactivity in screensharing: https://slackhq.com/screen-sharing-in-slack-made-interactive-cf8816efaa01

Seems to be quite difficult technically, so it’s only something to might think about in the future. But maybe it gets easier or there’s some open source libaries in the future.

**Comment by nickvergessen** [comment]
Other search terms: mouse pointer laser

**Comment by ruedigerkupper** [comment]
Zoom has this feature, but zoom is under intense discussion in the EU. Some schools have been forbidden to be using zoom and are in desperate need of alternatives.
This is the opportunity. If we are quick, we can fill this gap. But it must be in the next days or weeks for the EU to recognize Nextcloud as a valid alternative. Better days than weeks.

**Comment by ruedigerkupper** [comment]
See also comment in #1167.

**Comment by srkunze** [comment]
I completely agree. There are two features that are really crucial to help people and for meetings:

- whiteboard (emulated by sharing GIMP :D )
- highlight things on their screen (newbies, colleagues) to show them around THEIR environment

BBB has the whiteboard feature but no highlighting (really sucking: "go one line up, no not two, more to left, left, left, no the other left, right, right)

Both Zoom and Slack have the highlighting feature.

Jitsi doesn't have it at all.

**Comment by thomasesr** [comment]
Perhaps just a simple remote laser pointer would be enough for most cases. But I would love this feature for remote work collaboration.

**Comment by srkunze** [comment]
Yes. Still needed. :-)

It would already suffice when the audience could laser point to something in the browser window. This would already solve the issue for people with two monitors or when sharing single application windows.

It's highly unlike to have permission to laser-point on a random point of the screen from inside of a javascript application (not sure if there's even an API for this).

**Comment by JPustkuchen** [comment]
This would be suuuuuuuuch a great feature!
https://pop.com/ is also worth a look. It also works without installing the client, just in the browser.

Remote control should require an opt-in, while remote pointing might be enabled by default.

**Comment by bentuna** [comment]
We would really appreciate this feature. Can we help somehow to get it done?

**Comment by bentuna** [comment]
Maybe this is helpful: [Realtime collaborative drawing with canvas and WebRTC](https://dev.to/nyxtom/realtime-collaborative-drawing-with-canvas-and-webrtc-2d01)

**Comment by konradbr** [comment]
Bumping this topic. It's something that would be useful and some people I work with have me switch to Zoom _(hate it)_ for this very reason.

**Comment by sechanbask** [comment]
We want to quit Teams, and this feature is a must have for us.

**Comment by bentuna** [comment]
This feature would still be a gamechanger for many people to switch to Talk!

**Comment by pdmarf** [comment]
Desperately need a simple laser pointer to use when teaching students who are sharing their screen with me. Thanks for moving this to the design review!

**Comment by neufeind** [comment]
Related for full remote-control (remote-input, ...): #14280
